python/tests/types/test_vision.py

```diff
@@ -0,0 +1,51 @@
+#  Copyright 2021 Rikai Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import pytest
+
+from binascii import b2a_base64
+from PIL import Image as PILImage
+import numpy as np
+
+from rikai.types.vision import Image
+
+def test_show_embedded_png(tmp_path):
+    data = np.random.random((100,100))
```

Comment:
lol not like to use `unittest.TestCase`?

Reply:
yeah, makes the code simpler most of the time, but I'm open to discussion on testing convention.

Reply:
i am fine either way.  lets get a consistent style ?